python/sglang/srt/model_executor/cuda_graph_runner.py

```diff
@@ -162,6 +162,26 @@ def set_torch_compile_config():
     monkey_patch_torch_compile()
 
 
+def get_prefill_seqlen_to_capture(model_runner: ModelRunner):
+    server_args = model_runner.server_args
+    if server_args.cuda_graph_prefill_max_seqlen is None:
+        capture_prefill_seqlen = list(range(32, 1025, 32))
+    else:
+        capture_prefill_seqlen = list(
+            range(32, server_args.cuda_graph_prefill_max_seqlen + 1, 32)
+        )
+    compile_prefill_seqlen = (
+        [
+            # seqlen
+            # for seqlen in capture_prefill_seqlen
+            # if seqlen <= server_args.torch_compile_max_bs
```

Comment:
hmm getting confused about this... do you mean we do not do any compile, then maybe just write

```
compile_prefill_seqlen = [] # unsupported currently
```